asmcomp/arm/emit.mlp

```diff
@@ -572,19 +600,11 @@ let emit_instr i =
           1 + ninstr
         end
     | Lop(Iintop(Icomp cmp)) ->
-        let compthen = name_for_comparison cmp in
-        let compelse = name_for_comparison (negate_integer_comparison cmp) in
         `	cmp	{emit_reg i.arg.(0)}, {emit_reg i.arg.(1)}\n`;
-        `	ite	{emit_string compthen}\n`;
-        `	mov{emit_string	compthen}	{emit_reg i.res.(0)}, #1\n`;
-        `	mov{emit_string compelse}	{emit_reg i.res.(0)}, #0\n`; 4
+        emit_set_condition cmp i.res.(0) + 1
```

Comment:
I found the addition on this line (and the one below) confusing at first.  Maybe "`1 + emit_set_condition ...`" would be clearer?

Reply:
I hesitated between the two.  If you find `1 + ...` clearer, let's go for it.